account_payment_return/models/account_journal.py

```diff
@@ -9,9 +8,9 @@ class AccountJournal(models.Model):
     _inherit = "account.journal"
 
     default_expense_account_id = fields.Many2one(
-        comodel_name='account.account', string='Default Expense Account',
+        comodel_name='account.account', string='Default Commission Account',
```

Comment:
Why changing this? Commission is more for sales, not for the meaning we are thinking (comisión).

Reply:
I think both terms, bank expenses an bank commissions are right, I changed it because @elicoidal suggested it and I suppose he has better feeling than me about the best term in English language.

Reply:
@pedrobaeza @luismontalba I suggested `commission` instead of `expense` to avoid the confusion with standard Odoo `expense` account.
Alternative would `Charges` (this is what appears in my bank statement) or `Fees`

Reply:
I like Charges!

Reply:
OK, I'll put Charges, It's a good option